docs/internal/github_actions.rst

```diff
@@ -145,14 +145,128 @@ The last list file used by ``build_and_test.yml`` is :ghfile:`tools/modeling/tes
 
 To disable a test in GitHub Actions, ``!GH_ACTIONS`` must be added next to the test in the .lst file.
 These tests will not run when ``-Config GH_ACTIONS`` is passed alongside the lst file.
-There are similar test blockers which only block for specific github actions configurations: ``!GHA_OPENDDS_SAFETY_PROFILE`` blocks Safety Profile builds from running a test.
+There are similar test blockers which only block for specific github actions configurations from running marked tests:
+
+* ``!GH_ACTIONS_OPENDDS_SAFETY_PROFILE`` blocks Safety Profile builds
+
+* ``!GH_ACTIONS_M10`` blocks the MacOS10 runners
+
+* ``!GH_ACTIONS_ASAN`` blocks the Address Sanitizer builds
+
+* ``!GH_ACTIONS_W16`` blocks the Windows2016 runner
+
 These blocks are necessary because certain tests cannot properly run on GitHub Actions due to how the runners are configured.
 
 .. seealso::
 
   :doc:`running_tests`
     For how ``auto_run_tests.pl`` works in general.
 
+Blocked Tests
+=============
+
+Certain tests are blocked from GitHub actions because their failures are either unfixable, or are not represented on the
+scoreboard. If this is the case, we have to assume that the failure is due to some sort of limitation caused by the
+GitHub Actions runners.
```

Comment:
Can you make this and the rest of the rst changes [one line per sentence](https://opendds.readthedocs.io/en/latest/internal/docs.html#rst-sphinx-usage)?